source/containers.tex

```diff
@@ -9200,6 +9238,665 @@
+    constexpr reference       front();
+    constexpr const_reference front() const;
+    constexpr reference       back();
+    constexpr const_reference back() const;
+
+    // \ref{inplace.vector.data}, data access
+    constexpr       T* data()       noexcept;
+    constexpr const T* data() const noexcept;
+
+    // \ref{inplace.vector.modifiers}, modifiers
+    template<class... Args>
+      constexpr reference emplace_back(Args&&... args);                     // freestanding-deleted
+    constexpr reference push_back(const T& x);                              // freestanding-deleted
+    constexpr reference push_back(T&& x);                                   // freestanding-deleted
+    template<@\exposconcept{container-compatible-range}@<T> R>
+      constexpr void append_range(R&& rg);                                  // freestanding-deleted
+    constexpr void pop_back();
+
+    template<class... Args>
+      constexpr pointer try_emplace_back(Args&&... args);
+    constexpr pointer try_push_back(const T& x);
+    constexpr pointer try_push_back(T&& x);
+    template<@\exposconcept{container-compatible-range}@<T> R>
+      constexpr ranges::borrowed_iterator_t<R> try_append_range(R&& rg);
+
+    template<class... Args>
+      constexpr reference unchecked_emplace_back(Args&&... args);
+    constexpr reference unchecked_push_back(const T& x);
+    constexpr reference unchecked_push_back(T&& x);
+
+    template<class... Args>
+      constexpr iterator emplace(const_iterator position, Args&&... args);  // freestanding-deleted
+    constexpr iterator insert(const_iterator position, const T& x);         // freestanding-deleted
+    constexpr iterator insert(const_iterator position, T&& x);              // freestanding-deleted
+    constexpr iterator insert(const_iterator position, size_type n,         // freestanding-deleted
+                              const T& x);
+    template<class InputIterator>
+      constexpr iterator insert(const_iterator position,                    // freestanding-deleted
+                                InputIterator first, InputIterator last);
+    template<@\exposconcept{container-compatible-range}@<T> R>
+      constexpr iterator insert_range(const_iterator position, R&& rg);     // freestanding-deleted
+    constexpr iterator insert(const_iterator position,                      // freestanding-deleted
+                              initializer_list<T> il);
+    constexpr iterator erase(const_iterator position);
+    constexpr iterator erase(const_iterator first, const_iterator last);
+    constexpr void swap(inplace_vector& x)
+      noexcept(N == 0 || (is_nothrow_swappable_v<T> &&
+                          is_nothrow_move_constructible_v<T>));
+    constexpr void clear() noexcept;
+
+    constexpr friend bool operator==(const inplace_vector& x,
+                                     const inplace_vector& y);
+    constexpr friend synth-three-way-result<T>
```

Comment:
```suggestion
    constexpr friend @\exposid{synth-three-way-result}@<T>
```
Also, currently this will make the whole `inplace_vector<T, N>` ill-formed when `synth-three-way-result<T>` is ill-formed (see LWG4071), but an LWG issue seems needed.

Reply:
Thanks! fixed

Looking thru other occurrences of synth-three-way-result in containers.tex, some use `\exposid`, but most use `\placeholder`. They should all be `\exposid` right?

Reply:
Not sure why `\placeholder` was used in so many places. I think it's better to use `\exposid` for _`synth-three-way-result`_, see https://github.com/cplusplus/draft/wiki/Specification-Style-Guidelines#formatting.

Reply:
thanks - opened #7126; resolving here